rcl/test/CMakeLists.txt

```diff
@@ -62,14 +62,6 @@ function(test_target_function)
     AMENT_DEPENDENCIES ${rmw_implementation} "osrf_testing_tools_cpp" "test_msgs"
   )
 
-  rcl_add_custom_gtest(test_time${target_suffix}
-    SRCS rcl/test_time.cpp
-    ENV ${rmw_implementation_env_var} ${memory_tools_ld_preload_env_var}
-    APPEND_LIBRARY_DIRS ${extra_lib_dirs}
-    LIBRARIES ${PROJECT_NAME} osrf_testing_tools_cpp::memory_tools
-    AMENT_DEPENDENCIES ${rmw_implementation}
-  )
-
   rcl_add_custom_gtest(test_timer${target_suffix}
```

Comment:
what about the timer? i think `rcl` also deals with this too, but RMW implementation.

Reply:
The main reason I didn't change timer is because a timer instantiates a guard condition, which makes a call down into the RMW layer.  So theoretically there could be a bug in one of the RMW implementations, but not another, which this test would find.

Reply:
sounds reasonable. thanks!